packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -199,32 +160,131 @@ function RefreshButton(): React.ReactElement {
 	);
 }
 
+/**
+ * Props for {@link MenuSection}
+ */
+export type MenuSectionProps = React.PropsWithChildren<{
+	/**
+	 * Section header.
+	 */
+	header: React.ReactElement;
+}>;
+
+const useMenuSectionStyles = makeStyles({
+	root: {
+		display: "flex",
+		flexDirection: "column",
+	},
+});
+
 /**
  * Generic component for a section of the menu.
  */
 export function MenuSection(props: MenuSectionProps): React.ReactElement {
-	const { header, icon, children, onHeaderClick } = props;
-	const focusable = props.focusable === undefined ? false : true;
+	const { header, children } = props;
 
 	const styles = useMenuSectionStyles();
+
+	return (
+		<div className={styles.root}>
+			{header}
+			{children}
+		</div>
+	);
+}
+
+/**
+ * Props for {@link MenuSectionLabelHeader}
+ */
+export interface MenuSectionLabelHeaderProps {
+	/**
+	 * The text to display in header of the menu section.
+	 */
+	label: string;
+
+	/**
+	 * The icon to display in the header of the menu section.
+	 */
+	icon?: React.ReactElement;
+}
+
+const useMenuSectionLabelHeaderStyles = makeStyles({
+	root: {
+		alignItems: "center",
+		display: "flex",
+		flexDirection: "row",
+		fontWeight: "bold",
+	},
+});
+
+/**
+ * Simple menu section header with a label.
+ */
+export function MenuSectionLabelHeader(
+	props: MenuSectionLabelHeaderProps,
+): React.ReactElement {
+	const { label, icon } = props;
+	const styles = useMenuSectionLabelHeaderStyles();
+
+	return (
+		<div className={styles.root}>
+			{label}
+			{icon}
+		</div>
+	);
+}
+
+/**
+ * Props for {@link MenuSectionButtonHeader}
+ */
+export interface MenuSectionButtonHeaderProps extends MenuSectionLabelHeaderProps {
+	/**
+	 * Callback function that runs when the header is clicked.
+	 */
+	onClick?(): void;
+
+	/**
+	 * Button alt text.
+	 */
+	altText: string;
+}
+
+const useMenuSectionButtonHeaderStyles = makeStyles({
+	root: {
+		alignItems: "center",
+		display: "flex",
+		flexDirection: "row",
+		fontWeight: "bold",
+		cursor: "pointer",
+		role: "button",
```

Comment:
This is the key accessibility change. We need `role: "button"` to ensure navigation tools understand that the heading is selectable / clickable and serves the role of a button in the UX.